app/scripts/controllers/transactions/index.js

```diff
@@ -445,7 +445,7 @@ export default class TransactionController extends EventEmitter {
    * @param {string} editableParams.gasPrice
    * @returns {TransactionMeta} the txMeta of the updated transaction
    */
-  updateEditableParams(txId, { data, from, to, value, gas, gasPrice }) {
+  async updateEditableParams(txId, { data, from, to, value, gas, gasPrice }) {
```

Comment:
With this going async, is there anywhere we need to add an `await updateEditableParams (....`?

Reply:
Good question. Fortunately, it already has await in the one place where it would matter https://github.com/MetaMask/metamask-extension/blob/develop/ui/store/actions.js#L728